src/components/JobLogs/JobLogs.js

```diff
@@ -42,13 +48,25 @@ export class JobLogs extends Component {
 
     this.visibleColumns = [
       'fileName',
+      'log',
       'jobProfileName',
       'jobExecutionHrId',
       'completedDate',
       'runBy',
     ];
   }
 
+  createLogButton = id => (
```

Comment:
Move this logic to `withJobLogsCellsFormatter`. So `createLogButton` method is not needed in this file.